src/aleph_client/asynchronous.py

```diff
@@ -234,23 +254,49 @@ async def create_aggregate(
 
 async def create_store(
         account: Account,
-        address=settings.ADDRESS_TO_USE,
+        address: Optional[str] = None,
         file_content: Optional[bytes] = None,
+        file_path: Optional[str] = None,
         file_hash: Optional[str] = None,
-        guess_mime_type: bool = False,
+        guess_mime_type: Optional[bool] = None,
         ref: Optional[str] = None,
-        storage_engine=StorageEnum.storage,
+        storage_engine: Optional[StorageEnum] = None,
```

Comment:
Same here, it makes sense to use the `storage` engine by default.

Reply:
All these `None` are in place to comply better with mypy. The actual default is being set in the function body.

IMO the ideal behavior should be that if "nothing"/`None` is being passed to some argument with a default, then the default should instead be used, but Python overrides the default then with `None`.